operator/v1/register.go

```diff
@@ -34,8 +34,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(GroupVersion,
 		&Authentication{},
 		&AuthenticationList{},
-		&DNS{},
-		&DNSList{},
+		&DNSController{},
```

Comment:
/hold

I don't like `DNSController` - only ingress is called that, and that's because it's the term of art upstream.

Network is the same name as its operator and we aren't proposing renaming it.

Reply:
Okay, sounds like we'll want to be diligent in our docs to use qualified resource names. Thanks.